controllers/src/main/java/com/redhat/ipaas/controllers/NoopHandler.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.ipaas.controllers;
+
+import com.redhat.ipaas.model.integration.Integration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+import java.util.*;
+
+@Component
+@ConditionalOnProperty(value = "controllers.integration.enabled", havingValue = "noop")
```

Comment:
So what's the difference between noop and false? Maybe you can add some javadoc, and a paragraph to the readme?